### ISSUE TITLE
Is this still needed.

### ISSUE THREAD

**Issue by chriscoffee** [NHSLeadership/glusterfs-static-epel#1]
Are we cool to delete this epel or is it still required somewhere?

I'm assuming it was for the AMI's when we we using Gluster.

**Comment by cehwitham** [comment]
Yeah don't think we need this @chriscoffee. Was just used while we were using Gluster but believe everything is switched over to EFS now.
